### PR TITLE
Add Hong Jing's photo to About Us page

### LINKED ISSUE #8: Add Hong Jing's photo to About Us page
Add Hong Jing's photo to AboutUs.md

### PR DESCRIPTION
**Pull request by hj235**
Fixes #8

### CODE REVIEW

File: docs/images/hj235.png
Comment (karthickkc) [inline docs/images/hj235.png:1]: nice

Comment (karthickkc) [reply docs/images/hj235.png:1]: pls change the aboutUs doc

**Review by karthickkc — CHANGES_REQUESTED** [review]
Bad, please look at issue

**Review by hj235 — COMMENTED** [review]
PLEAS E REVIEW

**Review by karthickkc — APPROVED** [review]
LGTM!!!

**Comment by bbbbcccd** [comment]
💯

**Pull request merged**